src/service/performance-impl.js

```diff
@@ -97,7 +97,7 @@ export class Performance {
         });
 
     // Tick window.onload event.
-    loadPromise(win).then(() => {
+    loadPromise(win.document).then(() => {
```

Comment:
It appears to, [according](https://developer.mozilla.org/en-US/docs/Web/API/Document) to MDN, though it just points to the `window.onload` handler.